apps/src/util/formatValidation.js

```diff
@@ -5,3 +5,10 @@ export const isEmail = (value) => EMAIL_REGEX.test(value);
 
 const ZIP_CODE_REGEX = /^\d{5}([\W-]?\d{4})?$/;
 export const isZipCode = (value) => ZIP_CODE_REGEX.test(value);
+
+export const isInt = (value) => !window.isNaN(parseInt(value, 10))
```

Comment:
This is misleading as it will accept anything that *starts with* an integer, even if it's not entirely an integer.

```
> parseInt('55 and some other stuff', 10)
55

> parseInt('1.999', 10)
1
```

I'd expect isInt to ensure that the input is entirely an integer, not with additional decimals or text.